packages/api/src/routes/user.ts

```diff
@@ -158,23 +161,29 @@ router.get(
   })
 );
 
+const noAuthProviders = {
+  [PROVIDER_APPLE]: Apple,
+  [PROVIDER_TENOVI]: Tenovi,
+};
```

Comment:
Since the other related schemas are on `contants.ts`, maybe we should move this there as well?